pkg/api/login_oauth.go

```diff
@@ -179,3 +191,22 @@ func OAuthLogin(ctx *middleware.Context) {
 
 	ctx.Redirect(setting.AppSubUrl + "/")
 }
+
+func verifyScopes(got, want []string) error {
+	var missing []string
+
+loop:
```

Comment:
seems like just another name(syntax) for a goto :) 

Very ugly (and unintuitive) syntax as the label is outside the for loop.